migrations/20220225011500-create_user_types_table.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+const UserTypesSchema = require("../src/schema/UserTypes");
+const TABLE_NAME = "userTypes";
+module.exports = {
+  async up (queryInterface) {
+     return queryInterface.createTable(TABLE_NAME, UserTypesSchema).then(() => {
+       queryInterface.bulkInsert('userTypes', [{
+        type: "STUDENT"
+      }, {
+        type: "SCHOOL"
+      },{
+        type: "COMPANY"
```

Comment:
Can you make these as constants within the schema file and import them from there